src/pages/signin/SignInPage.js

```diff
@@ -154,6 +155,8 @@ function SignInPageInner({credentials, account, isInModal, activeClients, prefer
     const [hasInitiatedSAMLLogin, setHasInitiatedSAMLLogin] = useState(false);
 
     const isClientTheLeader = activeClients && ActiveClientManager.isClientTheLeader();
+    // eslint-disable-next-line rulesdir/no-negated-variables
+    const shouldShowAnotherLogin = Visibility.isVisible() && !isClientTheLeader;
```

Comment:
A brief comment explaining what this does would be useful I think as the name by itself isn't super clear!

Reply:
In fact I think we need to tweak the name, as it reads to me as more like "should we show another login form" (which obviously doesn't make sense!), what do you think about something like `hasAnotherLoginPageOpen`?

Reply:
Name updated and comment added